File: daytwo/python/afterbreak/school_grade.py
def Grade(scoreOne,scoreTwo,scoreThree): 
	summing = scoreOne + scoreTwo + scoreThree
	average_score = summing / 3
	
	if(90 <= average_score <= 100):
		letter_grade = "A"
	elif(80 <= average_score <= 90):
		letter_grade = "B"

	elif(70 <= average_score <= 80):
		letter_grade = "C"
		
	elif(60 <= average_score <= 70):	
		letter_grade = "D"	
		
	elif(0 <= average_score <= 60):	
		letter_grade = "F"

	return letter_grade 
	
	
	
	
	
scoreOne = int(input("enter score"))	
	
scoreTwo = int(input("enter score"))	
	
scoreThree = int(input("enter score"))	
	
grade = Grade(scoreOne,scoreTwo,scoreThree)	
	
print("the grade for the average of three scores is: ",grade)	
	
	
	
	
	
	

File: daytwo/java/afterbreak/SchoolGrade.java
import java.util.Scanner;
public class SchoolGrade{
	public static String grade(int scoreOne,int scoreTwo,int scoreThree){
	int sum = scoreOne + scoreTwo + scoreThree;
	int averageScore = sum / 3;
	
	String letterGrade = "e";
	
	if(averageScore > 90 && averageScore <= 100)
		letterGrade = "A";
		
	else if(averageScore > 80 && averageScore <= 90 )	
		letterGrade = "B";	
		
	else if(averageScore > 70 && averageScore <= 80)	
		letterGrade = "C";
		
	else if(averageScore >= 60 && averageScore <= 70)	
		letterGrade = "D";	
		
	else if(averageScore < 60)	
		letterGrade = "F";
		
		
return letterGrade;		
		
		
		
		
}

public static void main(String [] args){
Scanner input = new Scanner (System.in);

System.out.println("enter score: ");
int scoreOne = input.nextInt();


System.out.println("enter score");
int scoreTwo = input.nextInt();


System.out.println("enter score");
int scoreThree = input.nextInt();

String Grade = grade(scoreOne, scoreTwo, scoreThree);

System.out.println("grade is:" + Grade);




}


}
